shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.js

```diff
@@ -31,128 +31,163 @@ exports.fileCourtIssuedDocketEntryInteractor = async (
-      eventCode: documentMeta.eventCode,
-      filingDate: documentMeta.filingDate,
-      freeText: documentMeta.freeText,
-      isDraft: false,
-      isFileAttached: true,
-      isOnDocketRecord: true,
-      judge: documentMeta.judge,
-      numberOfPages,
-      scenario: documentMeta.scenario,
-      serviceStamp: documentMeta.serviceStamp,
-      trialLocation: documentMeta.trialLocation,
-      userId: user.userId,
-    },
-    { applicationContext },
-  );
+  const isUnservable = UNSERVABLE_EVENT_CODES.includes(documentMeta.eventCode);
 
-  const workItem = new WorkItem(
-    {
-      assigneeId: null,
-      assigneeName: null,
-      associatedJudge: caseToUpdate.associatedJudge,
-      caseStatus: caseToUpdate.status,
-      caseTitle: Case.getCaseTitle(caseEntity.caseCaption),
-      docketEntry: {
-        ...docketEntryEntity.toRawObject(),
-        createdAt: docketEntryEntity.createdAt,
-      },
-      docketNumber: caseToUpdate.docketNumber,
-      docketNumberWithSuffix: caseToUpdate.docketNumberWithSuffix,
-      hideFromPendingMessages: true,
-      inProgress: true,
-      section: DOCKET_SECTION,
-      sentBy: user.name,
-      sentByUserId: user.userId,
-    },
-    { applicationContext },
+  await Promise.all(
+    docketNumbers.map(async docketNumber => {
+      const caseToUpdate = await applicationContext
+        .getPersistenceGateway()
+        .getCaseByDocketNumber({
+          applicationContext,
+          docketNumber,
+        });
+
+      let caseEntity = new Case(caseToUpdate, { applicationContext });
+
+      const docketEntryEntity = new DocketEntry(
+        {
+          ...omit(subjectDocketEntry, 'filedBy'),
+          attachments: documentMeta.attachments,
+          date: documentMeta.date,
+          documentTitle: documentMeta.generatedDocumentTitle,
+          documentType: documentMeta.documentType,
+          draftOrderState: null,
+          editState: JSON.stringify(documentMeta),
+          eventCode: documentMeta.eventCode,
+          filingDate: documentMeta.filingDate,
+          freeText: documentMeta.freeText,
+          isDraft: false,
+          isFileAttached: true,
+          isOnDocketRecord: true,
+          judge: documentMeta.judge,
+          numberOfPages,
+          scenario: documentMeta.scenario,
+          serviceStamp: documentMeta.serviceStamp,
+          trialLocation: documentMeta.trialLocation,
+          userId: user.userId,
+        },
+        { applicationContext },
+      );
+
+      const workItem = new WorkItem(
+        {
+          assigneeId: null,
+          assigneeName: null,
+          associatedJudge: caseToUpdate.associatedJudge,
+          caseStatus: caseToUpdate.status,
+          caseTitle: Case.getCaseTitle(caseEntity.caseCaption),
+          docketEntry: {
+            ...docketEntryEntity.toRawObject(),
+            createdAt: docketEntryEntity.createdAt,
+          },
+          docketNumber: caseToUpdate.docketNumber,
```

Comment:
Small nit: Change `caseToUpdate` to `caseEntity`.  That way it uses the validated data.